### ISSUE TITLE
Window: GPU apps mess up mouse tracking

### ISSUE THREAD

**Issue by davidpanderson** [BOINC/boinc#2744]
On my Lenovo Win7 computer, when BOINC runs NVIDIA GPU apps, the system gets into a state where mouse tracking is messed up: every 5 or 10 seconds, the cursor freezes for a couple of seconds.  This persists even if I exit BOINC.  It goes away only on reboot.

I've noticed this with S@h and E@h GPU apps.

This might happen if the app get suspended while a kernel is in progress.  To prevent this, apps are supposed to put these in a critical section.  Are S@h and E@h doing this?

Note: my NVIDIA driver is old (dated 10/28/2013); the OS won't let me update it.  Maybe this doesn't happen with current drivers.

**Comment by RichardHaselgrove** [comment]
I've used NVidia cards under Windows (XP and 7) since January 2009, when we were debugging the NVidia-provided SETI app shortly after launch. I started with a 9800GT, and my best is currently a GTX 970: neither top of their respective ranges, but middling-good.

I've never noticed this particular problem. It could be the driver, but I'm usually slow to update my drivers too - I think I would have noticed, since I run SETI GPU even on my daily-use browser and email machine.

One possible thought: the current optimal SETI app is 8.22 (opencl_nvidia_SoG) which, as the plan class suggests, uses the OpenCL infrastructure. NVidia provides poor OpenCL support (they prefer people to use CUDA, but nobody has yet come up with an efficient implementation for VLAR on CUDA - that was the problem we had back in 2009). The 8.22 (opencl_nvidia_SoG) app runs best if an entire CPU core is kept free to support it, via app_config.xml: by default, BOINC tends to reserve 90-odd%, but that is insufficient for this particular app. That might be worth a try.
